db/seeds/muscle_group_descriptions.rb

```diff
@@ -0,0 +1,22 @@
+module MuscleGroupDescriptions
+  ABS =
+  "The abdominals are a stubborn muscle to train; moreover, due to the quick build up of lactic acid (burning feeling) during the exercises, most people avoid training them. It will be noted that all areas of the abdominal area must be trained to achieve fully developed the abs. The upper abs, lower abs, and middle abs along with the obliques must all be trained with various exercises. You do not need to train all four areas every single abdominal training session; however do not favor one area to another unless it is and area which is under developed. The proper way to train abdominals is with slow repetitions along with a controlled, strict range of motion. A true set of abdominals should only be between 12 to 15 repetitions, unless you are advanced and perform up to 20. If you are past this repetition range of 20, you are not training your abdominals effectively.You will notice this with individuals performing abdominal exercises of repetitions ranging from 50 and above; secondary muscles are applied. Abdominals recover quickly, so they may be trained every other day, perhaps every day by advanced individuals. Breathing air in and blowing it out is very important to the contraction and stretching of the abdominals during the exercises. As you contract your abdominal muscles, you need to blow out your air and contract, then when you release and stretch out, you need to breath air in. Performing this correctly will lead to results!"
+
+  ARMS =
+  "The muscles of the arms are one of the most sought after parts to develop. Your arm consists two thirds of triceps and one third of biceps, making the triceps muscle the larger of the two. Therefore, if you seek to gain more size in your arms, be sure to focus on the triceps to a greater degree. In men there is usually a strong emphasis placed on training the biceps over triceps. On the other hand, many women feel that they need to tone up th triceps area since over time, skin and fat begins to hang and look un-appealing with age. Toning up the triceps is the solution to this common problem. Complete arm development requires the training of both biceps and triceps."
```

Comment:
Line is too long. [677/80]<br>Indent the first line of the right-hand-side of a multi-line assignment.<br>Freeze mutable objects assigned to constants.